### DISCUSSION TITLE
Add author user to API for developers

### DISCUSSION THREAD

**Discussion by ramazaneris** [stoatchat/discussions#39] [Feature Suggestions]
## Add to author user to "<message>" parameters . return info who used command. like it : 
```js
<message>.author //return info who used command
<message>.author.username //return info the username of who used command
```

**Comment by sussycatgirl** [comment]
Not sure what you're trying to say?

**Reply by ramazaneris** [reply]
Like it discord.js 
Look discord.js

**Reply by abdulrahman1s** [reply]
@Jeosty take a look at [Better Revolt.JS](https://github.com/TheMaestro0/better-revolt.js).

**Comment by div2005** [comment]
Looks like getting information about the command sender 🤔

**Reply by ramazaneris** [reply]
yes , sure
